src/components/common/form/message-item-types/DoubleDateMoved.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import { screen, fireEvent, waitFor } from '@testing-library/react';
+import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
+import SingleDateMoved from './DoubleDateMoved';
+
+describe('DoubleDateMoved', () => {
+  it('should render a summary message with two date links', async () => {
+    renderWithTimecardContext(
+      <SingleDateMoved
```

Comment:
This implies it's rendering a SingleDateMoved component for the DoubleDateMoved test - looks like the import is wrong

Reply:
Updated typos